src/megachatapi_impl.cpp

```diff
@@ -341,9 +341,7 @@ void MegaChatApiImpl::sendPendingRequests()
         case MegaChatRequest::TYPE_LOGOUT:
         {
             bool deleteDb = request->getFlag();
-#ifndef KARERE_DISABLE_WEBRTC
```

Comment:
This `#ifndef` should be removed also from the `MegaChatRequest::TYPE_DELETE` case.